### PR TITLE
OADP-2157: Updating IBM Power and IBM Z systems section

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [OADP-2157](https://issues.redhat.com/browse/OADP-2157)

### VERSION:

* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16

### Link to docs preview:

* [OADP support for IBM Power and IBM Z](https://73935--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins#oadp-support-for-ibm-power-and-ibm-z)

### QE review:
- [X ] [QE has approved this change](https://github.com/openshift/openshift-docs/pull/73935#pullrequestreview-2017984573)

### CODE REVIEW

**Comment by stillalearner** [comment]
Latest release is OADP 1.1.7

Also please add versions 1.2.3 & 1.3.1 here.

File: modules/oadp-ibm-power-test-support.adoc [hunk 1]
```diff
@@ -6,4 +6,4 @@
 [id="oadp-ibm-power-test-matrix_{context}"]
 = OADP support for target backup locations using {ibm-power-title}
 
-{ibm-power-name} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
+{ibm-power-name} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.4 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and OADP 1.1.4 against all non-AWS S3 backup location targets as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-power-test-support.adoc:9]:
```suggestion
* {ibm-power-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.4 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.4 against all non-AWS S3 backup location targets as well.
* {ibm-power-name} running with {product-title} 4.12 and later, and {oadp-first} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.2.3 against all non-AWS S3 backup location targets as well.
* {ibm-power-name} running with {product-title} 4.12 and later, and {oadp-first} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.13 and later, and {oadp-short} 1.3.1 against all non-AWS S3 backup location targets as well.
```

File: modules/oadp-ibm-power-test-support.adoc [hunk 2]
```diff
@@ -6,4 +6,6 @@
 [id="oadp-ibm-power-test-matrix_{context}"]
 = OADP support for target backup locations using {ibm-power-title}
 
-{ibm-power-name} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
+* {ibm-power-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.4 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.4 against all non-AWS S3 backup location targets as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-power-test-support.adoc:9]:
```suggestion
* {ibm-power-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.7 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.7 against all non-AWS S3 backup location targets as well.
```

File: modules/oadp-ibm-z-test-support.adoc [hunk 3]
```diff
@@ -6,4 +6,4 @@
 [id="oadp-ibm-z-test-support_{context}"]
 = OADP testing and support for target backup locations using {ibm-z-title}
 
-{ibm-z-name} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
+{ibm-z-name} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.4 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and OADP 1.1.4 against all non-AWS S3 backup location targets as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-z-test-support.adoc:9]:
```suggestion
* {ibm-z-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.7 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.7 against all non-AWS S3 backup location targets as well.
* {ibm-z-name} running with {product-title} 4.12 and later, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.12 and later, and {oadp-short} 1.2.3 against all non-AWS S3 backup location targets as well.
* {ibm-z-name} running with {product-title} 4.13 and later, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.13 and later, and {oadp-short} 1.3.1 against all non-AWS S3 backup location targets as well.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 4]
```diff
@@ -21,7 +21,7 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
 
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-OADP 1.1.0 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.0 in terms of backup locations for these systems.
+OADP 1.1.4 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.4 in terms of backup locations for these systems.
```
Comment (stillalearner) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]: Here as well please.

Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]:
```suggestion
* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.7 in terms of backup locations for these systems.
* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.2.3 in terms of backup locations for these systems.
* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
```

**Review by stillalearner — APPROVED** [review]
lgtm

**Review by mramendi — COMMENTED** [review]
Thanks for all the work! Here go some suggestions

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 5]
```diff
@@ -21,7 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
 
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-OADP 1.1.0 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.0 in terms of backup locations for these systems.
+* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.7 in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:24]:
```suggestion
* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 6]
```diff
@@ -21,7 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
 
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-OADP 1.1.0 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.0 in terms of backup locations for these systems.
+* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
+* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.2.3 in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:25]:
```suggestion
* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 7]
```diff
@@ -21,7 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
 
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-OADP 1.1.0 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.0 in terms of backup locations for these systems.
+* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
+* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
+* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:26]:
```suggestion
* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.3.1 in terms of backup locations for these systems.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 8]
```diff
@@ -21,7 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
 
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-OADP 1.1.0 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.0 in terms of backup locations for these systems.
+* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
+* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:25]:
```suggestion
* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
```

File: backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc [hunk 9]
```diff
@@ -21,7 +21,9 @@ include::modules/oadp-supported-architecture.adoc[leveloffset=+1]
 
 OpenShift API for Data Protection (OADP) is platform neutral. The information that follows relates only to {ibm-power-name} and to {ibm-z-name}.
 
-OADP 1.1.0 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.1.0 in terms of backup locations for these systems.
+* {oadp-short} 1.1.7 was tested successfully against {product-title} 4.11 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.1.7 in terms of backup locations for these systems.
+* {oadp-short} 1.2.3 was tested successfully against {product-title} 4.12, 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for {oadp-short} 1.2.3 in terms of backup locations for these systems.
+* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13 and later for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc:26]:
```suggestion
* {oadp-short} 1.3.1 was tested successfully against {product-title} 4.13, 4.14, and 4.15 for both {ibm-power-name} and {ibm-z-name}. The sections that follow give testing and support information for OADP 1.3.1 in terms of backup locations for these systems.
```

File: modules/oadp-ibm-power-test-support.adoc [hunk 10]
```diff
@@ -6,4 +6,6 @@
...
 = OADP support for target backup locations using {ibm-power-title}
 
-{ibm-power-name} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
+* {ibm-power-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.7 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.7 against all non-AWS S3 backup location targets as well.
+* {ibm-power-name} running with {product-title} 4.12 and later, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.2.3 against all non-AWS S3 backup location targets as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-power-test-support.adoc:10]:
```suggestion
* {ibm-power-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.12, 4.13. 4.14, and 4.15, and {oadp-short} 1.2.3 against all non-AWS S3 backup location targets as well.
```

File: modules/oadp-ibm-power-test-support.adoc [hunk 11]
```diff
@@ -6,4 +6,6 @@
...
+* {ibm-power-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.7 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.7 against all non-AWS S3 backup location targets as well.
+* {ibm-power-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.12, 4.13. 4.14, and 4.15, and {oadp-short} 1.2.3 against all non-AWS S3 backup location targets as well.
+* {ibm-power-name} running with {product-title} 4.13 and later, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.13 and later, and {oadp-short} 1.3.1 against all non-AWS S3 backup location targets as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-power-test-support.adoc:11]:
```suggestion
* {ibm-power-name} running with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-power-name} with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 against all non-AWS S3 backup location targets as well.
```

**Comment by anarnold97** [comment]
@stillalearner - please can I ask you to double-check the changes I have made to this PR, and indicate if you are happy for me to push to **merge-review**?

Thanks

File: modules/oadp-ibm-z-test-support.adoc [hunk 12]
```diff
@@ -6,4 +6,6 @@
...
+* {ibm-z-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.7 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.7 against all S3 backup location targets, which are not AWS, as well.
+* {ibm-z-name} running with {product-title} 4.12 and later, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.12 and later, and {oadp-short} 1.2.3 against all S3 backup location targets, which are not AWS, as well.
+* {ibm-z-name} running with {product-title} 4.13 and later, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.13 and later, and {oadp-short} 1.3.1 against all S3 backup location targets, which are not AWS, as well.
```
Comment (stillalearner) [inline modules/oadp-ibm-z-test-support.adoc:11]: At other sections, you mentioned all the Ocp versions separately but not here? why diff format

Comment (anarnold97) [inline modules/oadp-ibm-z-test-support.adoc:11]:
```suggestion
* {ibm-z-name} running with {product-title} 4.13, 4.14, and 4.15, and {oadp-short} 1.3.1 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.13 4.14, and 4.15, and {oadp-short} 1.3.1 against all S3 backup location targets, which are not AWS, as well.
```

File: modules/oadp-ibm-z-test-support.adoc [hunk 13]
```diff
@@ -6,4 +6,6 @@
...
 = OADP testing and support for target backup locations using {ibm-z-title}
 
-{ibm-z-name} running with {product-title} 4.11 and 4.12, and OpenShift API for Data Protection (OADP) 1.1.2 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and OADP 1.1.2 against all non-AWS S3 backup location targets as well.
+* {ibm-z-name} running with {product-title} 4.11 and 4.12, and {oadp-first} 1.1.7 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.11 and 4.12, and {oadp-short} 1.1.7 against all S3 backup location targets, which are not AWS, as well.
+* {ibm-z-name} running with {product-title} 4.12 and later, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.12 and later, and {oadp-short} 1.2.3 against all S3 backup location targets, which are not AWS, as well.
```
Comment (anarnold97) [inline modules/oadp-ibm-z-test-support.adoc:10]:
```suggestion
* {ibm-z-name} running with {product-title} 4.12, 4.13, 4.14, and 4.15, and {oadp-short} 1.2.3 was tested successfully against an AWS S3 backup location target. Although the test involved only an AWS S3 target, Red Hat supports running {ibm-z-name} with {product-title} 4.12, 4.13, 4.14 and 4.15, and {oadp-short} 1.2.3 against all S3 backup location targets, which are not AWS, as well.
```

File: modules/oadp-ibm-z-test-support.adoc (hunk 12 above)
Comment (anarnold97) [reply modules/oadp-ibm-z-test-support.adoc]: Sorry, mistake

**Review by stillalearner — APPROVED** [review]
lgtm

**Pull request merged**